docs/tutorials/operations.md

```diff
@@ -0,0 +1,166 @@
+## Kubernetes
+
+In the beginning we chose to run Skipper as
+[daemonset](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)
+to run it on all worker nodes. Since 2018 we run Skipper as deployment
```

Comment:
What were the reasons for the change? I'm curious 😃

Reply:
We wanted to scale independently from nodes, because we can save costs or we need to scale more instances than we have nodes. Most likely the save costs case will happen, because skipper is quite performant compared to all the backends.

Reply:
Thanks for your explanation! That makes a lot of sense.